data-prepper-plugins/otel-trace-raw-prepper/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation "com.linecorp.armeria:armeria-grpc:1.5.0"
     implementation "com.fasterxml.jackson.core:jackson-databind:2.12.2"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.12.2"
+    implementation group: 'com.google.guava', name: 'guava', version: '11.0.2'
```

Comment:
Can we use a more recent version? - https://mvnrepository.com/artifact/com.google.guava/guava